src/components/FeedbackReportModal/FeedbackReportModal.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button } from '@components/Button'
+import { useImageLoadsSuccessfully } from '@lib/hooks/useLoadImage'
+import classNames from 'classnames'
+import useTranslation from 'next-translate/useTranslation'
+import { FC, Fragment } from 'react'
+import { Dialog, Transition } from '@headlessui/react'
+import Trans from 'next-translate/Trans'
+
+export interface FeedbackReportModalPropType {
+  title: string
+  address: string
+  treeName: string
+  imageUrl?: string
+  isOpen?: boolean
+  onConfirm?: () => void
+  onClose?: () => void
+}
+
+export const FeedbackReportModal: FC<FeedbackReportModalPropType> = ({
+  title,
+  address,
+  treeName,
+  imageUrl,
+  isOpen = true,
+  onConfirm = () => undefined,
+  onClose = () => undefined,
+}) => {
+  const { t } = useTranslation('common')
+  const isImageLoadable = useImageLoadsSuccessfully(imageUrl)
+  const showImg = imageUrl && isImageLoadable
+
+  const formattingComponents = {
+    bold: <strong />,
+  }
+
+  return (
+    <Transition appear show={isOpen} as={Fragment}>
```

Comment:
That's nice. I'll have to start looking into this transition helper more.

Reply:
I think so too! TBH I used this from headless UI examples. It's great!